### ISSUE TITLE
Bug python 3.4.X

### ISSUE THREAD

**Issue by marioidival** [Pylons/pyramid_mailer#67]
Hi, I tried send an email with pyramid_mailer using mailer.send_immediately(message_instance) I given this: *** TypeError: unorderable types: str() > int()

https://www.dropbox.com/s/rvxggi0wqdtdarl/Captura%20de%20tela%202015-05-21%2017.19.38.png?dl=0

**Comment by marioidival** [comment]
This bug is in stmp package of python 3.4.3

**Comment by mmerickel** [comment]
The issue appears to be that we are not properly converting the debug value into an `int` in `Mailer.from_settings`. This should be a simple fix if you want to do a pull request with a test. Otherwise someone else will get to it, I'm just not sure when.

**Comment by marioidival** [comment]
Yes! Fixed :dancer:

**Comment by marioidival** [comment]
```python
    @classmethod
    def from_settings(cls, settings, prefix='mail.'):
        """Create a new instance of 'Mailer' from settings dict.

        :param settings: a settings dict-like
        :param prefix: prefix separating 'pyramid_mailer' settings
        """
        settings = settings or {}

        kwarg_names = [prefix + k for k in (
                       'host', 'port', 'username',
                       'password', 'tls', 'ssl', 'keyfile',
                       'certfile', 'queue_path', 'debug', 'default_sender',
                       'sendmail_app', 'sendmail_template')]

        size = len(prefix)

        kwargs = dict(((k[size:], settings[k]) for k in settings.keys() if
                        k in kwarg_names))

        for key in ('tls', 'ssl'):
            val = kwargs.get(key)
            if val:
                kwargs[key] = asbool(val)

        for key in ('debug', 'port'):
            val = kwargs.get(key)
            if val:
                kwargs[key] = int(val)

        # list values
        for key in ('sendmail_template', ):
            if key in kwargs:
                kwargs[key] = aslist(kwargs.get(key))

        return cls(**kwargs)
````

**Comment by marioidival** [comment]
@mmerickel If I send the pull-request, how long will the new release coming out? (I'm on a task that goes into production in a few moments)

**Comment by mmerickel** [comment]
Fixed in #67.

**Comment by mmerickel** [comment]
I just released 0.14.1.
